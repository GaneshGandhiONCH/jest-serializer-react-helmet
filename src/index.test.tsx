import serializer from ".";

describe("Serialization module", () => {
  it("matches expectations", () => {
    expect(serializer).toMatchSnapshot();
  });
});
