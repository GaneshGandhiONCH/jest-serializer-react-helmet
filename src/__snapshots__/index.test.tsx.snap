// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Serialization module matches expectations 1`] = `
Object {
  "print": [Function],
  "test": [Function],
}
`;
